### PR TITLE
Added Quiz

### PR DESCRIPTION
**Pull request by marshalmiller**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #505 

Ref #

#### Changes done:
- [x] Added Quiz for Binary Cubes

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by marshalmiller** [comment]
For Issue #505. Binary Cubes Quiz

**Comment by manjotsidhu** [comment]
These are direct questions from the book, can we try to come up with better questions if possible ?

**Comment by marshalmiller** [comment]
@manjotsidhu I apologize. I assumed that was the desired outcome. I do not see any quiz-specific guidelines. Could you give me a better idea of where the assumed level of the user would be? If not gained from the book, what level of mastery or depth should these quizzes be at? I will revise accordingly.

File: docs/logic-design/binary-cubes.md [hunk 1]
```diff
@@ -60,3 +60,17 @@ The adjacency is key to group cubes in Karnaugh maps and in the Quine McCluskey
 ## References
 
 {% bibliography --cited  --file books %}
+
+{:.quiz}
+1. An Object with 1 variable is represented by which shape?
```
Comment (manjotsidhu) [inline docs/logic-design/binary-cubes.md:65]:
```suggestion
1. Which shape is represented by an object having exactly 1 variable?
```
We can improve these questions by re-phrasing them

**Comment by manjotsidhu** [comment]
Please also rephrase the other questions like we did for the first question.

**Comment by marshalmiller** [comment]
Good idea. I have rephrased them accordingly.

**Review by manjotsidhu — APPROVED** [review]
cc @Shivansh2407

**Pull request merged**